acceptance_tests/features/view_collection_exercises_data.feature

```diff
@@ -16,7 +15,7 @@ Feature: View Collection Exercise
     And the internal user can view all collection exercises for QBS
       | period | shown_to_respondent_as | status  |
       | 1803   | 9 March 2018           | Created |
```

Comment:
Looks like 1803 events are also being loaded as part of make setup. so need to change Created ==> Scheduled for period 1803.